test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/schedule/HapiScheduleDelete.java

```diff
@@ -0,0 +1,83 @@
+package com.hedera.services.bdd.spec.transactions.schedule;
+
+import com.google.common.base.MoreObjects;
+import com.hedera.services.bdd.spec.HapiApiSpec;
+import com.hedera.services.bdd.spec.transactions.HapiTxnOp;
+import com.hedera.services.bdd.spec.transactions.TxnUtils;
+import com.hedera.services.usage.schedule.ScheduleDeleteUsage;
+import com.hederahashgraph.api.proto.java.FeeData;
+import com.hederahashgraph.api.proto.java.HederaFunctionality;
+import com.hederahashgraph.api.proto.java.Key;
+import com.hederahashgraph.api.proto.java.ScheduleDeleteTransactionBody;
+import com.hederahashgraph.api.proto.java.Transaction;
+import com.hederahashgraph.api.proto.java.TransactionBody;
+import com.hederahashgraph.api.proto.java.TransactionResponse;
+import com.hederahashgraph.fee.SigValueObj;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.List;
+import java.util.function.Consumer;
+import java.util.function.Function;
+
+import static com.hedera.services.bdd.spec.transactions.TxnUtils.suFrom;
+
+public class HapiScheduleDelete extends HapiTxnOp<HapiScheduleDelete> {
+    static final Logger log = LogManager.getLogger(HapiScheduleDelete.class);
+
+    private String schedule;
+
+    @Override
+    public HederaFunctionality type() {
+        return HederaFunctionality.ScheduleDelete;
+    }
+
+    public HapiScheduleDelete(String schedule) {
+        this.schedule = schedule;
+    }
+
+    @Override
+    protected HapiScheduleDelete self() {
+        return this;
+    }
+
+    @Override
+    protected long feeFor(HapiApiSpec spec, Transaction txn, int numPayerKeys) throws Throwable {
+        return spec.fees().forActivityBasedOp(
+                HederaFunctionality.ScheduleDelete, this::usageEstimate, txn, numPayerKeys);
+    }
+
+    private FeeData usageEstimate(TransactionBody txn, SigValueObj svo) {
+        return ScheduleDeleteUsage.newEstimate(txn, suFrom(svo)).get();
+    }
+
+    @Override
+    protected Consumer<TransactionBody.Builder> opBodyDef(HapiApiSpec spec) throws Throwable {
+        var tId = TxnUtils.asScheduleId(schedule, spec);
+        ScheduleDeleteTransactionBody opBody = spec
+                .txns()
+                .<ScheduleDeleteTransactionBody, ScheduleDeleteTransactionBody.Builder>body(
+                        ScheduleDeleteTransactionBody.class, b -> {
+                            b.setScheduleID(tId);
+                        });
+        return b -> b.setScheduleDelete(opBody);
+    }
+
+    @Override
+    protected List<Function<HapiApiSpec, Key>> defaultSigners() {
+        return List.of(
+                spec -> spec.registry().getKey(effectivePayer(spec)));
```

Comment:
I think that you should add the `adminKey` as a default signer as-well, otherwise TX will revert by default